python/lsst/meas/base/forcedPhotCoadd.py

```diff
@@ -25,28 +26,29 @@
 import lsst.coadd.utils
 import lsst.afw.table
 
-from .forcedPhotImage import ProcessImageForcedConfig, ProcessImageForcedTask
+from .forcedPhotImage import ForcedPhotImageConfig, ForcedPhotImageTask
 
 __all__ = ("ForcedPhotCoaddConfig", "ForcedPhotCoaddTask")
 
-class ForcedPhotCoaddConfig(ProcessImageForcedConfig):
+
+class ForcedPhotCoaddConfig(ForcedPhotImageConfig):
     footprintDatasetName = lsst.pex.config.Field(
-        doc = ("Dataset (without coadd prefix) that should be used to obtain (Heavy)Footprints for sources."
-               "  Must have IDs that match those of the reference catalog."
-               "  If None, Footprints will be generated by transforming the reference Footprints."),
-        dtype = str,
-        default = "meas",
+        doc=("Dataset (without coadd prefix) that should be used to obtain (Heavy)Footprints for sources."
```

Comment:
I agree. I took a cleanup pass on the code to unify multi-line strings (as one commit) and change str % foo -> str % (foo,) as another commit.